wrappers/javascript/anoncreds-react-native/cpp/turboModuleUtility.cpp

```diff
@@ -160,9 +163,9 @@ FfiCredentialEntry jsiToValue(jsi::Runtime &rt, jsi::Object &options,
   if (value.isObject()) {
     jsi::Object valueAsObject = value.asObject(rt);
     auto credential = jsiToValue<ObjectHandle>(rt, valueAsObject, "credential");
-    auto timestamp = jsiToValue<int64_t>(rt, valueAsObject, "timestamp");
+    auto timestamp = jsiToValue<int64_t>(rt, valueAsObject, "timestamp", true);
     auto revocationState =
-        jsiToValue<ObjectHandle>(rt, valueAsObject, "revocationState");
+        jsiToValue<ObjectHandle>(rt, valueAsObject, "rev_state", true);
```

Comment:
Why are we suddenly using snake_case here? Is that because  we're mapping to what the anoncreds binary expects?

Reply:
I think it can remain being named `revocationState` because what anoncreds binary receives in the end is the FfiCredentialEntry structure.